StartBonusActivation/handler.ts

```diff
@@ -387,68 +353,63 @@ export function StartBonusActivationHandler(
         }))
       )
       .chain<BonusActivationWithFamilyUID>(({ dsu, familyUID }) => {
-        // this part is on a sub-chain as it handles the lock/unlock protection mechanism
-        // familyUID serves as lock context and thus is needed in scope for every sub-task
-        return (
-          taskEither
-            .of<StartBonusActivationResponse, void>(void 0)
-
-            .chain(_ =>
-              createBonusActivation(
-                bonusActivationModel,
-                fiscalCode,
-                familyUID,
-                dsu
-              )
-            )
-            .chain(bonusActivation =>
-              runStartBonusActivationOrchestrator(
-                client,
-                bonusActivation,
-                fiscalCode
-              ).map(_ => bonusActivation)
+        return taskEither
+          .of<StartBonusActivationResponse, void>(void 0)
+          .chain(_ =>
```

Comment:
This is useless, we can return directly createBonusActivation
```suggestion
        return createBonusActivation(
```